### PR TITLE
Deployment Manager: fix java namespace error

### PR DESCRIPTION
**Pull request by ChenTanyi**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/deploymentmanager/resource-manager/readme.md [hunk 1]
```diff
@@ -193,7 +193,7 @@ Please also specify `--azure-libraries-for-java=<path to the root directory of y
 
 ``` yaml $(tag) == 'package-2019-11-01-preview' && $(java) && $(multiapi)
 java:
-  namespace: com.microsoft.azure.management.deploymentmanager.v2019-11-01-preview
+  namespace: com.microsoft.azure.management.deploymentmanager.v2019_11_01_preview
   output-folder: $(azure-libraries-for-java-folder)/sdk/deploymentmanager/mgmt-v2019-11-01-preview
```
Comment (fengzhou-msft) [inline specification/deploymentmanager/resource-manager/readme.md]: Do you want to change the folder to `mgmt-v2019_11_01_preview` as well?

Comment (ChenTanyi) [reply specification/deploymentmanager/resource-manager/readme.md]: Yeah, I change it.

Comment (ChenTanyi) [reply specification/deploymentmanager/resource-manager/readme.md]: Done

**Review by fengzhou-msft — APPROVED** [review]

**Comment by fengzhou-msft** [comment]
@NullMDR who can help with the [Avocado error](https://dev.azure.com/azure-sdk/public/_build/results?buildId=212316&view=logs&j=f10f5372-40ed-54fc-caa4-b1cb5bd1252b&t=d3b2b841-a93b-5fa5-47c3-d8c4f6eae6e3)?

**Comment by NullMDR** [comment]
It's azure pipeline's bug that reports error status. This PR is ready to merge.

**Comment by fengzhou-msft** [comment]
> It's azure pipeline's bug that reports error status. This PR is ready to merge.

Do you have the permission to merge when required CI does not pass? I don't have the permission to do it.

**Pull request merged**